exonum-java-binding-fakes/src/main/java/com/exonum/binding/fakes/mocks/UserServiceAdapterMockBuilder.java

```diff
@@ -36,7 +36,29 @@
 @SuppressWarnings({"unused", "WeakerAccess"}) // Used in native code
 public final class UserServiceAdapterMockBuilder {
 
-  private final UserServiceAdapter service = mock(UserServiceAdapter.class);
+  private final UserServiceAdapter service;
+
+  /**
+   * Default constructor.
```

Comment:
Please remove this comment — it has negative value. Could probably make the class and ctor package-private to avoid having to document it.

Reply:
Ok, I added it just because of Checkstyle warnings. Going to make it package-private.